### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by orweis**
Two new open-source packages on top of FastAPI for Websocket-RPC and Pub/Sub

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,6 +108,8 @@
 - [FastAPI ServiceUtils](https://github.com/skallfass/fastapi_serviceutils) - Generator for creating API services.
 - [FastAPI SocketIO](https://github.com/pyropy/fastapi-socketio) - Easy integration for FastAPI and SocketIO.
 - [FastAPI Utilities](https://github.com/dmontagu/fastapi-utils) - Reusable utilities: class-based views, response inferring router, periodic tasks, timing middleware, SQLAlchemy session, OpenAPI spec simplification.
+- [⚡ FastAPI Websocket Pub/Sub 🗞️](https://github.com/authorizon/fastapi_websocket_pubsub) - The classic pub/sub pattern made easily accessible and scalable over the web and across your cloud in realtime. FastAPI + WebSockets + PubSub ==  💪 ❤️
```
Comment (Kludex) [inline README.md:111]:
```suggestion
- [FastAPI Websocket Pub/Sub](https://github.com/authorizon/fastapi_websocket_pubsub) - The classic pub/sub pattern made easily accessible and scalable over the web and across your cloud in realtime.
```

File: README.md [hunk 2]
```diff
@@ -108,6 +108,8 @@
...
+- [⚡ FastAPI Websocket RPC](https://github.com/authorizon/fastapi_websocket_rpc) - RPC (bidirectional JSON RPC) over Websockets made easy, robust, and production ready.
```
Comment (Kludex) [inline README.md:112]:
```suggestion
- [FastAPI Websocket RPC](https://github.com/authorizon/fastapi_websocket_rpc) - RPC (bidirectional JSON RPC) over Websockets made easy, robust, and production ready.
```

**Review by Kludex — CHANGES_REQUESTED** [review]
This PR is probably not going to be accepted with the emojis. The maintainer already expressed his feelings about them on a previous PR. :kissing: 

Do you mind removing them?

Thank you for the contribution! :smile:

**Comment by orweis** [comment]
Hi @mjhea0, 
The public has called these packages awesome - "This is awesome, much gratitude for this!"
https://www.reddit.com/r/FastAPI/comments/lpriit/fastapi_websocket_rpc_and_pubsub_packages/
Any chance we can move forward with the PR ?

**Pull request merged**